packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -114,13 +115,38 @@ class SideNavItem extends SideNavChildrenMixin(DisabledMixin(ElementMixin(Themab
         reflectToAttribute: true,
       },
 
+      /**
+       * Whether to use exact matching when comparing the item's path with the
+       * current browser URL, or not. `true` by default.
+       *
+       * With exact matching, the item is considered current only when the paths
+       * are exactly the same. With non-exact matching, the item is considered
+       * current when the item's path is a prefix of the pathname of the current
+       * browser URL. For example, with exact matching, an item with the path
+       * `/path` is considered current only when the browser URL is `/path`.
+       * With non-exact matching, an item with the path `/path` is considered
+       * current when the browser URL is `/path`, `/path/child`,
+       * `/path/child/grandchild`, etc.
+       *
+       * Note that this only affects matching of the URLs path, not the base
+       * origin or query parameters.
+       *
+       * @type {boolean}
+       */
+      matchExact: {
+        type: Boolean,
+        value: true,
+      },
+
       /**
        * Whether the item's path matches the current browser URL.
        *
        * A match occurs when both share the same base origin (like https://example.com),
        * the same path (like /path/to/page), and the browser URL contains at least
        * all the query parameters with the same values from the item's path.
        *
+       * See the `exact` property for how to change the path matching behavior.
```

Comment:
Should this be `matchExact`?
Also, can be changed to the link using the following syntax:

```
[`matchExact`](#/elements/vaadin-side-nav-item#property-matchExact)
```

Reply:
Done